src/kOS/Screen/Interpreter.cs

```diff
@@ -64,6 +64,10 @@ public override bool SpecialKey(char key)
             if (key == (char)UnicodeCommand.BREAK)
             {
                 Shared.Cpu.BreakExecution(true);
+                LineBuilder.Remove(0, LineBuilder.Length); // why isn't there a StringBuilder.Clear()?
```

Comment:
I was confused because there is one. then i realized it isnt available in our ancient version of c#